src/main/java/uk/ac/sanger/sccp/stan/service/work/WorkSummaryServiceImp.java

```diff
@@ -13,15 +16,22 @@
 @Service
 public class WorkSummaryServiceImp implements WorkSummaryService {
     private final WorkRepo workRepo;
+    private final WorkTypeRepo workTypeRepo;
 
-    public WorkSummaryServiceImp(WorkRepo workRepo) {
+    public WorkSummaryServiceImp(WorkRepo workRepo, WorkTypeRepo workTypeRepo) {
         this.workRepo = workRepo;
+        this.workTypeRepo = workTypeRepo;
     }
 
     @Override
-    public Collection<WorkSummaryGroup> loadWorkSummary() {
+    public WorkSummaryData loadWorkSummary() {
+        List<WorkType> workTypesList = new ArrayList<>();
+        Iterable<WorkType> workTypes = workTypeRepo.findAll();
```

Comment:
`List<WorkType> workTypes = BasicUtils.asList(workTypeRepo.findAll());`

Reply:
Ah nice will do that. I was trying to find a similar way to coerce the iterable to a list but couldnt find one 👍